src/ocean/text/json/Json.d

```diff
@@ -31,60 +31,64 @@ import ocean.text.json.JsonParser;
 
 import Float = ocean.text.convert.Float;
 
-version(UnitTest) import ocean.core.Test;
+version (UnitTest)
+{
+    import ocean.core.Test;
+    import ocean.io.Stdout;
+}
+
+///
+unittest
+{
+    void example ( )
```

Comment:
I am not sure how exactly comments as unit tests work but do you need to actually run `example()`?

Reply:
Nope, usage examples (`///` followed by `unittest`) don't need to be run, they just need to be compiled.